src/rpc/blockchain.cpp

```diff
@@ -1136,7 +1136,7 @@ UniValue getblockchaininfo(const JSONRPCRequest& request)
             "  \"verificationprogress\": xxxx, (numeric) estimate of verification progress [0..1]\n"
             "  \"chainwork\": \"xxxx\"     (string) total amount of work in active chain, in hexadecimal\n"
             "  \"pruned\": xx,             (boolean) if the blocks are subject to pruning\n"
-            "  \"pruneheight\": xxxxxx,    (numeric) lowest-height complete block stored\n"
+            "  \"pruneheight\": xxxxxx,    (numeric) lowest-height complete block stored (if pruning is enabled)\n"
```

Comment:
Update to `(only present if pruning is enabled)`. See [this example](https://github.com/bitcoin/bitcoin/blob/791a0e6ddade27d1b69f4861a6640de60b9553cf/src/wallet/rpcwallet.cpp#L1845) for reference.